### PR TITLE
Add January 2022 Community Spotlight Update to Cadence blog

### PR DESCRIPTION
**Pull request by sharanf**
¨

### CODE REVIEW

**Review by demirkayaender — APPROVED** [review]

**Review by emrahs — APPROVED** [review]

File: src/blog/_posts/2022-01-31-community-spotlight-january-2022.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Below are a selection of Cadence related articles, blogs and whitepapers. Please take a look and fee free to share via your own social media channels.
+
+- [Meet Cadence: Workflow Engine for Taming Complex Processes](https://thenewstack.io/meet-cadence-workflow-engine-for-taming-complex-processes/)
+- [Spinning Your Workflows with Cadence](https://www.instaclustr.com/blog/spinning-your-workflows-with-cadence/)
+- [Instaclustr Joins the Engineering Team at Uber in Supporting Cadence, the Powerful Open Source Orchestration Engine](https://www.globenewswire.com/news-release/2021/12/07/2347314/0/en/Instaclustr-Joins-the-Engineering-Team-at-Uber-in-Supporting-Cadence-the-Powerful-Open-Source-Orchestration-Engine.html)
+
+## Upcoming Events 
+
+- [Webinar : What is Cadence? And is it right for you?](https://info.instaclustr.com/webinar-emea-what-is-cadence.html?_ga=2.191041518.510582234.1643223308-2138855655.1638190316)</li>
```
Comment (just-at-uber) [inline src/blog/_posts/2022-01-31-community-spotlight-january-2022.md:41]: the trailing </li> element should be removed.

Comment (sharanf) [reply src/blog/_posts/2022-01-31-community-spotlight-january-2022.md]: Thanks! I've removed it now.

**Review by just-at-uber — APPROVED** [review]
You will need to sign the Contributor License Agreement to be able to merge this change (see waiting build step).

**Comment by sharanf** [comment]
> You will need to sign the Contributor License Agreement to be able to merge this change (see waiting build step).

I have done already signed the CLA. I think the issue is the that in the first commit I had a typo in my email address so it didnt recognise my github id. I've fixed the email for the other commits. When I actually click the 'details' link about the CLA and sign in with Github - it says you have already agreed the CLA. Not sure what to do to fix it.

**Comment by just-at-uber** [comment]
ok i can manually merge if thats the case. thank you.

**Comment by sharanf** [comment]
> ok i can manually merge if thats the case. thank you.

@just-at-uber excellent and thanks very much! :-)

**Pull request merged**